sass/directives/_flyout.scss

```diff
@@ -87,3 +87,122 @@
     }
   }
 }
+
+// Use the mobile-subnav mixin for the subnav bars that appear on our subscription / product pages
+// Displays a single link with a flyout
+@mixin mobile-subnav-flyout() {
+  @media only screen and (max-width: $small-screen-max) {
+    &:hover {
+      ul {
+        display: block;
+      }
+
+      .mobile-subnav__top-link {
+        &:after {
+          content: '';
+        }
+      }
+    }
+
+    ul {
+      display: none;
+      background-color: white;
+      border-bottom: $base-border;
+      position: absolute;
+      top: $product_subnav--height;
+      left: 0;
+      right: 0;
+      width: 100%;
+
+      > li {
+        display: block;
+        line-height: $product_subnav--height;
+        width: 100%;
+        
+        &:hover {
+          background-color: $base-link-color;
+        }
+
+        a {
+          display: block;
+          float: none;
+          border: none;
+          color: $primary-nav-link-color;
+          margin: 0 $container-edge-padding;
+          
+          &.active {
+            border-bottom: none;
+          }
+
+          &:hover {
+            color: white;
+          }
+        }
+      }
+    }
+  }
+
+  .mobile-subnav__top-link {
+    display: block;
+
+    @media only screen and (min-width: $medium-screen-min) {
+      display: none;
+    }
+  }
+
+  .mobile-subnav__top-link {
+    line-height: $product_subnav--height;
+    color: $primary-nav-link-color;
+    padding-left: $container-edge-padding;
+    text-transform: uppercase;
+    position: relative;
+
+    &:after {
+      font-family: FontAwesome;
+      font-size: $top-bar-angle-size;
+      color: $base-link-color;
+      content: '\f107';
+      position: absolute;
+      right: $container-edge-padding;
+    }
+  }
+
+  .mobile-subnav__close {
+    display: none;
+    
+    &:after {
+      font-family: FontAwesome;
+      font-size: $top-bar-angle-size;
+      color: $base-link-color;
+      content: '\f107';
```

Comment:
In the revised version in 1359, the new hover state for the initial link Susan requested includes unicode for the opposite angle.